ext/native/gfx_es2/gpu_features.cpp

```diff
@@ -189,6 +194,13 @@ void CheckGLExtensions() {
 			// They weren't, reset to GLES 2.0.
 			gl_extensions.ver[0] = 2;
 			gl_extensions.ver[1] = 0;
+		} else if (parsed[0] && (gl_extensions.ver[0] != parsed[0] || gl_extensions.ver[1] != parsed[1])) {
+			// Something going wrong. Possible bug in GL ES drivers. See #9688
+			ILOG("GL ES version mismatch. Version string '%s' parsed as %d.%d but API return %d.%d. Fallback to GL ES 2.0.", 
+				versionStr ? versionStr : "N/A", parsed[0], parsed[1], gl_extensions.ver[0], gl_extensions.ver[1]);
+			
+			gl_extensions.ver[0] = 2;
+			gl_extensions.ver[1] = 0;
```

Comment:
I think we should report this situation (`#include "Core/Reporting.h"`, [`Reporting::ReportMessage("foo: %s", "bar");`](https://github.com/hrydgard/ppsspp/blob/master/Core/Reporting.h#L69).)  I'd hate to accidentally bump people down because of a weird parsing error, and not even know.

It might also help us know if this is happening on more x86 devices.

-[Unknown]